### ISSUE TITLE
sparse_softmax_cross_entropy_with_logits wrong annotation

### ISSUE THREAD

**Issue by zishuaiz** [tensorflow/tensorflow#13576]
https://github.com/tensorflow/tensorflow/blob/107cc777af7880c140d089e44ad898a6ba929286/tensorflow/python/ops/nn_ops.py#L1661

It should be `If logits are scalars (need to have rank >= 1) or if the rank
      of the labels is not equal to the rank of the logits minus one.`

**Comment by asimshankar** [comment]
Thanks for pointing that out, will send a fix.
